src/index.html

```diff
@@ -11,6 +11,36 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+
+    <form
+     class="search-bar search-bar_big"
+     data-qa= "big"
```

Comment:
an unnecessary space between `data-qa=` and `"big"`

Reply:
done! :)